list.txt

```diff
@@ -21065,6 +21067,7 @@ gloriousfuturedays.com
 gloservma.com
 glovebranders.com
 glovesprotection.info
+glowarena.com
```

Comment:
https://verifymail.io/domain/glowarena.com => it looks like it's safe

Reply:
Thanks for warning. Removed 👍